codewof/programming/serializers.py

```diff
@@ -0,0 +1,27 @@
+"""Serializers for programming models."""
+
+from rest_framework import serializers
+from programming.models import Question
+
+
+class QuestionSerializer(serializers.ModelSerializer):
+    """Serializer for codeWOF questions."""
+
+    question_type = serializers.SerializerMethodField(read_only=True)
+
+    def get_question_type(self, obj):
```

Comment:
We can change the parameter name to whatever we want, so might as well have `question` instead of `obj`. Removes comment on line 15 also.